modules/nf-core/hicexplorer/hicpca/tests/main.nf.test

```diff
@@ -0,0 +1,58 @@
+nextflow_process {
+
+    name "Test Process HICEXPLORER_HICPCA"
+    script "../main.nf"
+    process "HICEXPLORER_HICPCA"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "hicexplorer"
+    tag "hicexplorer/hicpca"
+
+    test("mm9 chr1") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file('https://github.com/deeptools/HiCExplorer/raw/master/hicexplorer/test/test_data/hicPCA/mm9_reduced_chr1.cool', checkIfExists: true)
```

Comment:
shall we copy the data?

Reply:
I wonder if this is actually too big to be effective test data, at least it seems slow on this tool.

Reply:
then subset and upload?

Reply:
Uhh, I have absolutely no idea what a cool file is. I just want to migrate pytests to nf-test ;)

Reply:
ok, let's add it to the TODO

Reply:
Not like there aren't many other modules with links to other test-datasets (or external)